src/webapp/components/form/validations.ts

```diff
@@ -0,0 +1,29 @@
+import { ValidationErrorKey } from "../../../domain/entities/ValidationError";
+import { FieldType } from "./FormFieldsState";
+
+export function validateFieldRequired(value: any, fieldType: FieldType): ValidationErrorKey[] {
```

Comment:
Probably no need to change it, but here happens what we say in other code, as we have value and the fieldType isolated, TS cannot help us with the types.